src/main/native/jgskit.mac.mak

```diff
@@ -86,19 +98,20 @@ headers :
 		--add-exports java.base/sun.security.util=ALL-UNNAMED \
 		-d ${JAVACLASSDIR} \
 		-h ${TOPDIR}/src/main/native/ \
-		${TOPDIR}/src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java \
 		${TOPDIR}/src/main/java/com/ibm/crypto/plus/provider/ock/FastJNIBuffer.java \
+		${TOPDIR}/src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java \
 		${TOPDIR}/src/main/java/com/ibm/crypto/plus/provider/ock/OCKContext.java \
 		${TOPDIR}/src/main/java/com/ibm/crypto/plus/provider/ock/OCKException.java
 
 endif # ! EXTERNAL_HEADERS
 
 clean :
```

Comment:
Why separate these targets? If so shouldnt various calling scripts such as buildNative.sh etc, and other makefiles all follow suite? Seems more simple to just have one clean... ? I did notice also that `jgskit.mak` is also doing this ( and the clean and cleanAll targets not called )

Reply:
I did it to make it similar to `jgskit.mak`. You are right. I'll need to add it to the shell script. Or I can revert it to what it was. What do you think is best?

Reply:
Well i prefer to keep a simple `clean` that deletes all built artifacts in one shot. That is unless there is a good reason not to :) Id vote to revert this update and also rever the `jgskit.mak` back to use a single `clean` target.

Reply:
LGTM